src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -42,6 +50,83 @@ public static Index parseIndex(String oneBasedIndex) throws ParseException {
         return Index.fromOneBased(Integer.parseInt(trimmedIndex));
     }
 
+    /**
+     * Parse a series of indexes in a String into a Set of Index.
+     * index in the String can be spaced apart anyhow and if the index
+     * is invalid etc; Not a number, throws a MESSAGE_INVALID_INDEX
+     *
+     * @param indexes series of indexes to be parsed
+     * @return a set of Index
+     * @throws ParseException if invalid index is found
+     */
+    public static Set<Index> parseAttendees(String indexes) throws ParseException {
+        HashSet<Index> indexSet = new HashSet<>();
+        String[] arrayOfIndexes = indexes.split("\\s+"); //if indexes are randomly spaced apart
+
+        Arrays.stream(arrayOfIndexes).forEach(index -> {
+            try {
+                indexSet.add(parseIndex(index.trim())); //creates and add indexes to the hashset
+            } catch (ParseException parseException) {
+                //invalid index
```

Comment:
I think we can store the invalid indexes (so that we can display them to the user), and continue adding the rest of the valid indexes to the meeting. We should also consider what happens when we create a meeting with only invalid indexes for attendees: 
1. Is no meeting created at all?
2. Or is a meeting created with no attendees?  

I would prefer the second alternative, since meetings can be edited and deleted.